### PR TITLE
Install .v files

### LINKED ISSUE #17159: Coq stdlib no longer has installed .v files
I'm not completely sure I've accurately diagnosed the problem, but it seems that the build artifacts from CI no longer(?) include `.v` files installed alongside `.vo` and `.glob` files in `_install_ci/lib/coq/theories/`, which means that coqbot CI minimization cannot inline the standard library (as desired at https://github.com/coq/coq/pull/16967#issuecomment-1348525437, cc @SkySkimmer)

### PR DESCRIPTION
**Pull request by Alizter**
It would be nice if we could squeeze this into 8.17. cc @Zimmi48 

Fixes https://github.com/coq/coq/issues/17159. See there for context.

### CODE REVIEW

**Review by JasonGross — APPROVED** [review]
Thank you!

**Comment by Alizter** [comment]
@coqbot run full ci

**Comment by Alizter** [comment]
Here you can see the .v files are installed:
https://gitlab.com/coq/coq/-/jobs/3657700557/artifacts/browse/_install_ci/lib/coq/theories/Arith/

File: tools/dune_rule_gen/coq_module.ml [hunk 1]
```diff
@@ -82,11 +82,13 @@ let native_install_files ~tname ~rule coq_module =
 let base_install_files ~rule coq_module =
   match rule with
```
Comment (ejgallego) [inline tools/dune_rule_gen/coq_module.ml:83]:
```suggestion
  mod_to_obj coq_module ~ext:".v" :: 
  match rule with
```

**Comment by Alizter** [comment]
Again for reference, the .v files being installed:
```
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Evar.vos
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Float.glob
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Float.v
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Float.vo
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Float.vos
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Fresh.glob
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Fresh.v
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Fresh.vo
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Fresh.vos
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ident.glob
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ident.v
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ident.vo
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ident.vos
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ind.glob
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ind.v
Installing /builds/coq/coq/_install_ci/lib/coq/user-contrib/Ltac2/Ind.vo
```

**Review by ejgallego — APPROVED** [review]
Thanks!

**Comment by ejgallego** [comment]
Obvious fix, merging now given the impact on the minimizer.

@coqbot: merge now

**Pull request merged**